### PR TITLE
RHDEVDOCS-6053: Content creation for GitOps 1.11.5 RN

### PR DESCRIPTION
**Pull request by Dhruv-Soni11**
Version(s):

GitOps 1.11 and GitOps 1.12

Issue:

https://issues.redhat.com/browse/RHDEVDOCS-6053

Link to docs preview:

Release Notes change: https://75960--ocpdocs-pr.netlify.app/openshift-gitops/latest/release_notes/gitops-release-notes.html#release-notes-for-gitops-1-11-5_gitops-release-notes

SME review: @reginapizza
QE review: @varshab1210
Internal peer review: @eromanova97
Peer review: @dfitzmau 

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

**Review by varshab1210 — COMMENTED** [review]
Left a minor comment, LGTM otherwise

File: modules/gitops-release-notes-1-11-5.adoc [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+{gitops-title} 1.11.5 is now available on {OCP} 4.12, 4.13, and 4.14.
+
+[id="errata-updates-1-11-5_{context}"]
+== Errata updates
+
+[id="rhsa-XXXX-NNNN-gitops-1-11-5-security-update-advisory_{context}"]
+=== RHSA-2024:131902-04 - {gitops-title} 1.11.5 security update advisory
+
+Issued: 2024-05-15
```
[the remark this answers is not in the document]
Comment (Dhruv-Soni11) [reply modules/gitops-release-notes-1-11-5.adoc]: Fixed.

File: modules/gitops-release-notes-1-11-5.adoc [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+{gitops-title} 1.11.5 is now available on {OCP} 4.12, 4.13, and 4.14.
+
+[id="errata-updates-1-11-5_{context}"]
+== Errata updates
+
+[id="rhsa-2024-3475-gitops-1-11-5-security-update-advisory_{context}"]
+=== RHSA-2024:3475 - {gitops-title} 1.11.5 security update advisory
+
+Issued: 2024-05-28
```
Comment (varshab1210) [inline modules/gitops-release-notes-1-11-5.adoc:18]:
```suggestion
Issued: 2024-05-29
```

**Comment by varshab1210** [comment]
LGTM, thanks

**Comment by dfitzmau** [comment]
Hi @Dhruv-Soni11 . A rebase and push might be needed to correct the branch from being out od date.

File: modules/gitops-release-notes-1-11-5.adoc [hunk 3]
```diff
@@ -0,0 +1,34 @@
+// Module included in the following assembly:
+//
+// * release_notes/gitops-release-notes.adoc
+
+:_mod-docs-content-type: REFERENCE
+
```
Comment (dfitzmau) [inline modules/gitops-release-notes-1-11-5.adoc:6]: Not a big deal AFAIA, but the space is not necessary. Best to keep consistent with the example in the doc_guidelines.adoc file.

**Comment by dfitzmau** [comment]
One more comment but the rebase and push is needed befroe merging.

/remove-label peer-review-in-progress

/label peer-review-done

File: modules/gitops-release-notes-1-11-5.adoc (hunk 3 above)
Comment (Dhruv-Soni11) [reply modules/gitops-release-notes-1-11-5.adoc]: Thanks for letting me know about this @dfitzmau. I will implement it in the future as well. Fixed the feedback now.

**Comment by Dhruv-Soni11** [comment]
> One more comment but the rebase and push is needed befroe merging.
> 
> /remove-label peer-review-in-progress
> 
> /label peer-review-done

Rebased the branch and pushed the changes as well. Please confirm once the build is ready @dfitzmau. Thanks for the prompt review.

**Comment by dfitzmau** [comment]
LGTM

**Review by skopacz1 — APPROVED** [review]
Merge review LGTM

**Pull request merged**